src/components/ProductCard/Productcard.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { Link } from 'react-router-dom';
+import { FaStar,FaRegHeart,FaRupeeSign } from "react-icons/fa";
+import "../ProductCard/Productcard.css";
+
+
+export const Productcard = ({userProduct}) => {
+  return (
```

Comment:
I would suggest you to destructure userProduct here.
```
const {isSoldOut, brand, productName} = userProduct
```

Reply:
will do